OPHD/States/MapViewStateTurn.cpp

```diff
@@ -295,14 +295,23 @@ void MapViewState::updateResources()
 			std::clamp(stored.resources[3], 0, 25)
 		};
 
-		auto newResources = mPlayerResources + moved;
-		auto capped = newResources.cap(mRefinedResourcesCap / 4);
+		auto& storageTanksList = NAS2D::Utility<StructureManager>::get().structureList(Structure::StructureClass::Storage);
+		for (auto storageTanks : storageTanksList)
+		{
+			if (moved.empty()) { break; }
```

Comment:
This is checked inside the loop because the moved resources could be empty before the end of the storage list is reached (potentially far before hand depending on the number of structures).

Reply:
Ahh, right, that makes sense. I forgot that value is updated as the list is traversed.

I suppose the name is a little ambiguous about whether it's the amount to try and move, or the amount that was successfully moved.